config/db_config.yml.example

```diff
@@ -1,13 +1,13 @@
-session_secret: <%= ENV["SYSOPIA_SESSION_SECRET"] %>
-timezone_offset: <%= ENV["SYSOPIA_TIMEZONE_OFFSET"] %>
+session_secret: changeme
+timezone_offset: 0
 database:
   common: &common
     adapter: mysql2
     encoding: utf8
     reconnect: true
-    host: <%= ENV["SYSOPIA_DB_HOST"] %>
```

Comment:
We use Docker to deploy Sisopia for EOL production, this is the reason behind public config file with 'secret' ENV variables. It is also a pattern that becomes used often in RoR community -- public configs with private ENV vars